profiles/base.yaml

```diff
@@ -298,8 +298,8 @@ cambrinth_cap: 32
 stored_cambrinth: false
 # used with use_auto_mana. Multiplies your discern limit by this number to get a safe mana amount.
 prep_scaling_factor: 0.85
-# used with use_auto_mana. How often it will check discerns. Lower circle/ranks should check more often.
-check_discern_timer: 24
+# used with use_auto_mana. How often it will check discerns in hours. Lower circle/ranks should check more often.
```

Comment:
This is a merge issue. This and like 302.

Reply:
@Hazado can you merge master into your branch to make sure this is resolved?